src/main/scala/devices/tilelink/DevNull.scala

```diff
@@ -52,6 +53,6 @@ abstract class DevNullDevice(params: DevNullParams, beatBytes: Int, device: Simp
       alwaysGrantsT      = params.acquire)),
     beatBytes  = beatBytes,
     endSinkId  = if (params.acquire) 1 else 0,
-    minLatency = 1))) // no bypass needed for this device
+    minLatency = params.minLatency))) // TODO: does this comment still apply? no bypass needed for this device
   val tl_xing = this.crossIn(node)
```

Comment:
minLatency is NOT a free parameter. It describes the implementation.

Reply:
this PR changes it so that Error device can change its min latency to either 1 or 0.

Reply:
@terpstra can you please re-review. I have changed it so that the parameter is `bufferError`. But i do still think that DevNullDevice needs to know what minLatency it actually is. Hardcoding it as 1 here seems incorrect. I removed the comment since I wasn't sure what it was trying to say.

Reply:
Yes, the other changes seem to make this fine.